src/main/java/com/zendesk/maxwell/util/AbstractConfig.java

```diff
@@ -95,6 +100,29 @@ protected Properties readPropertiesFile(String filename, Boolean abortOnMissing)
 		return p;
 	}
 
+	protected Properties readPropertiesEnv(String envConfig) {
+		LOGGER.debug("Attempting to read env_config param: {}", envConfig);
+		String envConfigJsonRaw = System.getenv(envConfig);
+		if (envConfigJsonRaw != null && envConfigJsonRaw.startsWith("{")) {
+			LOGGER.debug("Parsing envConfig");
```

Comment:
probably more useful to log each key/value pair to the debug log than to say this

Reply:
I will log the keys, we should avoid logging the values which might contain passwords to the logs, even in debug mode. Went ahead and added this elsewhere too be consistent.